test/test_katello/test_deb_tracer.py

```diff
@@ -0,0 +1,73 @@
+import unittest2 as unittest
+
+# sys.path.append(os.path.join(os.path.dirname(__file__), '../../src/'))
+# from katello.tracer import upload_tracer_profile, query_affected_apps
+# from katello.constants import YUM, ZYPPER
+from katello.deb_tracer import needrestart, has_needrestart
+import subprocess
+
+from mock import patch, Mock
+
+import pytest
```

Comment:
While I like pytest, this appears to be unused.

Reply:
Pytest is used in this line https://github.com/Katello/katello-host-tools/blob/de6121ea1935f531c903a71695feb67fcc518828/test/test_katello/test_deb_tracer.py#L57

Reply:
Please use [`assertRaises`](https://docs.python.org/dev/library/unittest.html#unittest.TestCase.assertRaises) from `unittest` instead. I'm not opposed to using pytest (it's an amazing framework), but I'd like to avoid using two styles.